### ISSUE TITLE
Multimedia data license

### ISSUE THREAD

**Issue by pemensik** [raceintospace/raceintospace#420]
It was mentioned originally in issue #203, that RIS already had [attempt to add it into Debian](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=761636). But reason for refusal was unclear license for multimedia data.

Leon @peyre , you have been around Race into Space for ages, according to [webarchive](https://web.archive.org/web/20070810200921/http://www.geocities.com/raceintospace/). But Debian reviewer noted [copyright claim of some authors](https://web.archive.org/web/20070710110823/http://www.geocities.com/raceintospace/CopyrightInfo.txt), which would unfortunately mean the game data are not free software.

Do you know, remember or can find any relevant history, how the game data were made available? We have got code released under GNU GPL 2. But have copyright holder authorized also data distribution?

Debian reviewers mentioned data/audio/cd_track_*.ogg. They are quite good, but according to my check they never were used by the game. Their author is known, but AFAIK there is no evidence he agreed to publish it.

We may need to move multimedia without clear license to separate repository, which would not be shipped in release. That would make the game free software without problems with packaging and distribution. Do you have any contact to original copyright holders? Was there difference between released floppy and CD version, regarding to data?

**Comment by peyre** [comment]
The game's copyright reverted to Fritz Bronner and Michael McCarty, and they opted to open-source it.  However, some of the code for the multiplayer options wasn't open-source, so they disabled it.  This is a copy of an IM session I had with Michael about it back in 2014:
`(08:03:34 PM) LordPeyre: Hey Michael, got a quick question: why was the Play by Email option disabled in RIS?  Did it not work properly?
(08:04:14 PM) Michael McCarty: yeah.  There was a serial library that we didn't have rights to redistribute
(08:04:29 PM) LordPeyre: Oh!  So it's a copyright-related issue?
(08:04:38 PM) Michael McCarty: yeah
(08:04:45 PM) Michael McCarty: it shouldn't be too tough to add back in
(08:04:45 PM) LordPeyre: D'oh!  That does throw a wrench in the works.
(08:04:50 PM) LordPeyre: oh?
(08:04:54 PM) Michael McCarty: it's just serial code
(08:05:32 PM) Michael McCarty: the bsd socket code is all you really need
(08:05:51 PM) Michael McCarty: you don't need serial code, just use tcp/ip.
(08:06:13 PM) LordPeyre: Adding a TCP/IP option to RIS would be way cool too, though more ambitious
(08:06:30 PM) Michael McCarty: it's a way to get play by email going
(08:06:50 PM) Michael McCarty: you'll just connect to the SNMP sever and send the file over
(08:06:58 PM) LordPeyre: Oh, I see
(08:07:05 PM) Michael McCarty: not SNMP, I meant SMTP
(08:07:15 PM) LordPeyre: Oops--yes, of course.
(08:07:24 PM) LordPeyre: Email, not network monitoring
(08:07:31 PM) Michael McCarty: yep
(08:07:45 PM) LordPeyre: So, it would be PBEM with automatic sending?
(08:07:52 PM) LordPeyre: That would be neat.
(08:07:59 PM) Michael McCarty: it could be
(08:08:13 PM) Michael McCarty: or just launch any email client with the file to send
(08:08:55 PM) Michael McCarty: something in the host OS will handle the mailto: and pass that off to the email program of choice
(08:09:50 PM) LordPeyre: Ah, ok
(08:10:49 PM) Michael McCarty: it the games don't last too long you could just connect to someone and send the turn data back and forth
`

**Comment by rnyoakum** [comment]
@peyre - The PBEM code mentioned got removed, so that's fine. The issue concerns the data files - with extra emphasis on video and audio files (such as the music).

**Comment by peyre** [comment]
Oh that's good to hear.  Unfortunately I don't have anything for you about the AV files.  I could ask Michael about them, but I need more specific information about what to ask.

**Comment by pemensik** [comment]
There are only notes from Fritz  Bronner and Michael  McCarty as authors of the game. But it is not very clear whether they were copyright owners for sound and video files too. Were they releasing it as Strategic Visions representatives, who held all copyrights?

Since name of Brian Langsbard is mentioned in audio CD rip tracks, has he agreed to redistribution of his work? Or was he employee or contractor of Fritz or Michael, so they have redistribution rights for his work?

In other words, is only the game code open source or are all game data open as well?

**Comment by peyre** [comment]
Brian Langsbard's name is on the MP3s because the disk cover for the BARIS CD lists him in the credits for the musical score.  I filled in that information myself when I ripped the tracks from the CD.

The copyright reverted to Fritz and Michael by both an expiration date and Interplay's bankruptcy.  I don't know the details, but this is what Michael wrote about it:

Digging through my garage today I found the publishers contract we had with Interplay.  According to this all of Interplay"s rights expired 3 years after publishing or 6 months after actively marketing it, whichever was later.  This puts the contract expiration was near 1996-97 and the full rights of everything belonging to Fritz and myself.  There was also a clause where any bankruptcy by Interplay would have reverted all rights back to us as well.

**Comment by hweimer** [comment]
A related question is whether new multimedia content has been added that was not included in the CD version (e.g., the videos mentioned in #112)? If yes, can this content be distributed under the terms of the GPL?

**Comment by peyre** [comment]
As far as I know, all of RIS's multimedia comes from the CD version of the game.

**Comment by Symbian9** [comment]
Small question:

*Is it possible to use* `Race Into Space - Intro Music.mp3` *&* `Race Into Space - Final Music` *as background music in videos for YouTube?*

- https://sourceforge.net/projects/raceintospace/files/

**Comment by peyre** [comment]
I don't see why not.  Those are tracks on the original BARIS CD that I ripped to MP3.  Fritz and Michael have okayed the distribution of the CD as long as no one profits from it, so I'd think the two music tracks would be included in that.

**Comment by Symbian9** [comment]
> Fritz and Michael have okayed the **distribution of the CD as long as no one profits from it**, so I'd think the two music tracks would be included in that.

Does it mean, that at least licensing on *music tracks* is clear here & it could be used under CretiveCommons `CC-BY-NC-SA 4.0` License?

- https://creativecommons.org/licenses/by-nc-sa/4.0/

**Comment by hweimer** [comment]
I am not a copyright lawyer and I do not hold any copyright to the MP3 files in question (only Fritz Bronner and Michael McCarty do). However, it is widely held that distributing a copyrighted work (i.e., the original game) and creating derivative works (what you want to do) are different actions under copyright law requiring separate permission from the copyright holders.

On the other hand, you could use the MP3 files under the terms of the GNU GPL. There is some ambiguity whether the MP3s are distributed under the same version as the code (v2 or later), but one can probably assume that they are. However, this means that you would have to distribute your video under the terms of the GPL as well, which also requires you to supply the source files to create the video, whatever that means in your case. In general, most people avoid to use GPLed works for anything else but code because it can be quite impractical.

**Comment by pemensik** [comment]
Hello @mkmccarty, would it be possible to make some public statement about data license? Code is licensed under GPLv2 and you have committed to it. Because you were the last person from Strategic Visions, Inc. involved in the development, you might be the best representative.

Could be data license changed to something usable to content? Creative commons variant would be the best. [CC-BY-SA](http://creativecommons.org/licenses/by-sa/4.0/) would be the best option. As free software definition it would be allowed in all Linux distributions, both [Fedora](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses_3) and [Debian](https://wiki.debian.org/DFSGLicenses#Creative_Commons_Attribution_Share-Alike_.28CC-BY-SA.29_v4.0) allows it.
Non-commercial only license would make the game non-free, Creative Commons Attribution-NonCommercial-ShareAlike is [BAD in Fedora](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Bad_Licenses_3). Fedora for example does not allow non-free content, the game would have to be removed from it. As Hendrik pointed above, GNU GPL is impractical for multimedia data.

Could you or Fritz Bronner assign Creative Commons variant to game's data? Are ripped music tracks under the same license? Because I think @peyre commited them, it would be nice to have your explicit approval here. Leon Baradat's word is not enough for some distributions to accept the game into them.

Thank you.

**Comment by peyre** [comment]
I emailed Michael directly in case he didn't get your message above, and he wrote back.  Not good news, I think.

> These license things are sticky, entirely with the music, both the CD audio and midi files.  While the rights to the audio is owned outright by Fritz and myself with no royalties due to the composer or musicians, Fritz is more passionate about the rights aspect of this and I don't want to violate that.   It's this reason that I won't change the data license for the music.  Fritz may want to use the compositions in future versions or adaptations of Liftoff 2 or other purposes.

It would be better if someone created something new to use and swap that out so that music can be under whatever license that creator wanted to use.

I'm aware that there have been Android versions which were sold, but I'm not going to pursue anything related to that, the game is ages old and any fans of it may want to purchase Fritz's new version of the board game.  This part of it is a good thing.

**Comment by Symbian9** [comment]
> It looks like we may need to look into creating our own intro and end music for the game.

Maybe, there is already good music in the OpenGameArt Music repo:

- https://opengameart.org/art-search-advanced?keys=&field_art_type_tid[]=12&sort_by=created&sort_order=DESC

Just use search for `space`, `intro`, `flight`, etc.

**Comment by peyre** [comment]
Wow, that's a great resource!  I'll have to look through it sometime.

**Comment by pemensik** [comment]
Not that bad in the end. What we require is clear licensing terms. If only music and midi files would still require non-commercial use only, we can make them just addition to the basic game. If the game would still be playable without music, it can be distributed in all free software distributions with clear license. Music files would be separate addon with different license. For [Fedora](http://fedoraproject.org) distribution, it would make raceintospace game to be shipped without music files. And music files package could be shipped via [rpmfusion](http://rpmfusion.org), stating non-commercial use only.

I think it is better than uncertainity, especially if the music is still allowed under some conditions. @peyre, it would be still good to have clear license to at least video, images and gamedata. Video and pictures are in general low-resolution to be reused for any new project I think. Can you ask Michael for clarification, whether music would still be allowed at least under [CC BY-NC 4.0](https://creativecommons.org/licenses/by-nc/4.0/)? Or any other variant? Can we move it to separate repository with specific terms?

It would still be great, if original rights owners made statement for every (sub)directory under [data](https://github.com/raceintospace/raceintospace/tree/master/data), stating under what license is that directory considered. Even if I would miss original music in common distribution, at least playable game is still better than nothing.

We should move any content without Free Culture License to separate archive. Should we move or delete music files now?

**Comment by mkmccarty** [comment]
I'll spend some time thinking about this over the weekend.  I'll make a
statement for the data directories so their use is clear.

I'll read more about CC BY-NC 4.0.  Is there a license whereby the music
can only be used with this project?

Thanks for the interest in wanting to make this more easily distributed in
rpm form.

**Comment by peyre** [comment]
Gaa, I just hit Send on an email to you Michael.  Please ignore my missive - you seem to be already on top of this one.

**Comment by Symbian9** [comment]
> I'll read more about CC BY-NC 4.0. Is there a license whereby the music can only be used with this project?

Probably `CC BY-NC-ND 4.0`, but there should be a clear statement that those music files are non-separable part of the "Race Into Space" software.

- https://creativecommons.org/about/cclicenses/
  - https://creativecommons.org/licenses/by-nc-nd/4.0/

**Comment by peyre** [comment]
Speaking of licensing issues, should we remove the copyright symbol from the bottom left of the splash screen (like so)?
![first img 2](https://user-images.githubusercontent.com/3118014/103444763-abcdfc80-4c20-11eb-8fd8-9a76025b4ccc.png)

**Comment by pemensik** [comment]
I know only one project with separate licensing for content. [Planeshift](https://www.planeshift.it/) project had GPL code base but content under [no-reuse license](https://www.planeshift.it/License). It should be possible to make similar license for our music data. But we have nothing like official server, so it would have to be adjusted somehow.

**Comment by peyre** [comment]
I emailed Michael asking about whether we should leave the copyright symbol or take it out (I don't know what copyright law says about these things), he wrote back, saying "Maybe it gets changed and nobody notices, possibly replaced with some statement about OSS."  So I don't know where that leaves us.

**Comment by mkmccarty** [comment]
Generally copyrights should stay in place.  I don't recall what aspects
that copyright covers.  It could be the art, music and source code, not the
title or the game mechanics.

It's probably best to leave the original copyright and add an appropriate
line that clarifies the parts that were moved to CC at some point.   Are
there examples of this kind of thing?

**Comment by peyre** [comment]
Ah!!  Thanks for the clarification Michael!  Good, it's best to leave alone then; that makes it simple.

Incidentally, I emailed Fritz as you suggested, but haven't heard back yet.  He's probably in the middle of things.

**Comment by peyre** [comment]
You know, if the copyright holder isn't responding, I'm inclined to think we should continue with the game as it was when he and Michael open-sourced the game on SourceForge.  Further waiting, or further emails, are unlikely to get us answers in the foreseeable future.  If he was ok with the music as it was being used in RIS 1.1, I don't see why we can't leave it the way it is until we hear otherwise.

**Comment by hweimer** [comment]
I think we should also move the music files in data/audio/music/ into a separate repository to make clear that they are not available under the GPL. The game runs fine without them, although it might be a good idea to downgrade the severity of the error message from WARNING to INFO.

**Comment by peyre** [comment]
Has anyone worked out how to extract the MIDI intro and end music from the old floppy version?

**Comment by peyre** [comment]
Thanks to Michael, the midis are identified.  They're already extracted into data/midi as liftoff.mid and victory.mid.

**Comment by hweimer** [comment]
@peyre:
> Has anyone worked out how to extract the MIDI intro and end music from the old floppy version?

The ogg files used by the game have always been derived from the MIDI files and not from the orchestral version.

**Comment by peyre** [comment]
Thanks Hendrik.  Fortunately we have them saved and identified so we can just use them if we want to switch out the orchestral music for copyright purposes.
